### DISCUSSION TITLE
editor.value 에 관한 문의

### DISCUSSION THREAD

**Discussion by parkjh97** [dineug/erd-editor#348] [Q&A]
안녕하세요 api 문서를 통해 
editor.value 로 현재 editor의 value를 확인하는 법을 봤습니다.

혹시 마우스 우클릭했을때의 import => json 처럼 

특정 함수를 실행했을때 현재 editor.value 를 임의의 json으로 바꿔서 html에 표출하는 방법이 있을까요?

**Comment by dineug** [comment]
editor에 다른 정보를 불러오는 방법은 아래의 2가지로 제공하고 있습니다
- https://docs.erd-editor.io/ko/docs/api/erd-editor-element#setter
- https://docs.erd-editor.io/ko/docs/api/erd-editor-element#setinitialvalue

**Comment by parkjh97** [comment]
항상 좋은 답변 감사합니다. 
더불어 한가지 더 문의 사항이 있습니다.

외래키를 지정할때 현재 점선으로만 표기되는것으로 파악되는데 혹시 실선으로도 관계를 맺을수 있는 방법이 있을까요 또한
관계 지정시 선의 색상을 개별적으로 지정하는 방법도 같이 문의 드립니다.

**Reply by dineug** [reply]
외래키가 PK면 식별자로 실선이 됩니다.
선의 색은 개별 컬러를 지원하지 않고 있습니다
